src/clj_jargon/init.clj

```diff
@@ -27,8 +28,9 @@
   retval)
 
 (defn proxy-input-stream
-  [{^IRODSFileSystem cm-proxy :proxy} ^InputStream istream]
-  (let [with-jargon-index curr-with-jargon-index]
+  [cm ^InputStream istream]
+  (let [with-jargon-index curr-with-jargon-index
+        cm-proxy (if (delay? cm) (:proxy @cm) (:proxy cm))]
```

Comment:
(avoiding an NPE when it tries to close stuff below, but this time I'm forcing it to definitely be dereferenced... which shouldn't be an issue since it shouldn't come up unless it's open anyway)